operators/churner/churner.go

```diff
@@ -242,7 +247,11 @@ func (c *churner) getOperatorsToChurn(ctx context.Context, quorumIDs []uint8, op
 		// stake.
 		if new(big.Int).Mul(lowestStake, bipMultiplier).Cmp(new(big.Int).Mul(totalStake, churnBIPsOfTotalStake)) >= 0 {
 			c.metrics.IncrementFailedRequestNum("getOperatorsToChurn", FailReasonInsufficientStakeToChurn)
-			return nil, api.NewInvalidArgError(fmt.Sprintf("operator to churn out must have less than %f%% of the total stake. Stake of the operator to churn: %d, total stake in quorum: %d, quorum ID: %d", float64(operatorSetParams.ChurnBIPsOfTotalStake)/100.0, lowestStake, totalStake, quorumID))
+			msg := "operator to churn out must have less than %f%% of the total stake. " +
+				"Block number used for this decision: %d, operatorId of the operator " +
+				"to churn: %x, stake of the operator to churn: %d, total stake in " +
+				"quorum: %d, quorum ID: %d"
+			return nil, api.NewInvalidArgError(fmt.Sprintf(msg, float64(operatorSetParams.ChurnBIPsOfTotalStake)/100.0, currentBlockNumber, lowestStakeOperatorId, lowestStake, totalStake, quorumID))
```

Comment:
It will be cleaner to print `lowestStakeOperatorId.Hex()`

Reply:
ok, didn't realize OperatorID has this formatting method.